admin-frontend/src/pages/Status.js

```diff
@@ -221,20 +299,32 @@ class Status extends React.Component {
     successTooltip = false,
     successLabel = "Configured",
     warningLabel = "Not configured",
+    opacity = false,
+    image = null
   }) {
     const tooltipId = label.replace(" ", "");
     return (
       <div>
-        <div style={{ display: "flex", marginBottom: 3 }}>
-          <h5>
-            {label}: <pre>{value}</pre>{" "}
-          </h5>
-          <div style={{ marginTop: -3, marginLeft: 6 }} id={tooltipId}>
-            <Badge color={success ? "success" : error ? "danger" : "warning"}>
-              <span>
-                {success ? successLabel : error ? error : warningLabel}
-              </span>
-            </Badge>
+        <hr />
+        <div className="row" style={{ display: "flex", marginBottom: 3}}>
+          <div className="col-8">
+            {image && <img
+                style={{ height: 24}}
+                alt=""
+                src={image}
+                className="mr-3"
+                />}
+                  <h5 style={{
+                    display: "inline-block",
+                    color: "#404043", 
+                    fontSize: opacity ? 16 : 18, 
+                    opacity: opacity ? 0.75 : 1, 
+                    fontFamily: "'Source Sans Pro', sans-serif"}}>
+                    {label}: <pre>{value}</pre>{" "}
+                  </h5>
+          </div>
+          <div className="col" style={{ marginTop: -3 }} id={tooltipId}>
+            <StatusBadge error={error} status={success ? label == "Initial setup" ? "success" : "configured" : error ? "danger" : "warning"} />
```

Comment:
use `successLabel` and `warningLabel` for this. in case of error, render the error.